### PR TITLE
Added ariaRequired inputs for radio group & file attachment/drop components

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

**Comment by Blackbaud-TrevorBurch** [comment]
@Blackbaud-AlexKingman just for being super sure of everything, should we have a `toBeAccessible` test for when aria-required is enabled for these?

**Comment by Blackbaud-AlexKingman** [comment]
@Blackbaud-TrevorBurch 

> @Blackbaud-AlexKingman just for being super sure of everything, should we have a `toBeAccessible` test for when aria-required is enabled for these?

It's a good thought, but the axe-core test doesn't know weather something should be required or not. It gives a false positive when the `aria-required` attribute is removed, so I'm not sure what good it would do us to have that extra test?

File: src/app/public/modules/file-attachment/file-attachment.component.ts [hunk 1]
```diff
@@ -78,6 +78,9 @@ export class SkyFileAttachmentComponent implements ControlValueAccessor, AfterVi
   @Input()
   public acceptedTypes: string;
 
+  @Input()
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/file-attachment/file-attachment.component.ts]: For the `ariaRequired` pull request (forms), is there ever a situation where we’d want the field to be "required" but not `aria-required`? I think that the source of truth should be the `required` input, and the component should automatically set the `aria-required` attribute based on that? https://github.com/blackbaud/skyux-forms/blob/master/src/app/public/modules/file-attachment/file-attachment.component.ts#L108

**Comment by Blackbaud-AlexKingman** [comment]
Closing in favor of adding `required` inputs to each form component.

**Pull request closed without merging**